src/test/java/com/flightstats/hub/webhook/TimedWebhookStrategyTest.java

```diff
@@ -2,13 +2,26 @@
 
 import com.flightstats.hub.model.ContentKey;
 import com.flightstats.hub.model.SecondPath;
+import com.flightstats.hub.test.Integration;
 import org.joda.time.DateTime;
 import org.joda.time.DateTimeZone;
+import org.junit.BeforeClass;
 import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import static org.junit.Assert.assertEquals;
 
 public class TimedWebhookStrategyTest {
+    @BeforeClass
+    public static void setupIntegration() {
+        Logger logger = LoggerFactory.getLogger(TimedWebhookStrategyTest.class);
+        try {
+            Integration.startAwsHub();
+        } catch (Exception e) {
```

Comment:
Same comment as above. We'll want to re-throw the exception so test execution doesn't continue.